src/test/util_tests.cpp

```diff
@@ -728,43 +728,63 @@ bool IsStringTrueBadArgTest(const char *arg1)
 
 BOOST_AUTO_TEST_CASE(util_Logging)
 {
-    {
-        using namespace Logging;
-        BOOST_CHECK_EQUAL(8, sizeof(categoriesEnabled));
-        BOOST_CHECK_EQUAL(NONE, categoriesEnabled);
-        LogToggleCategory(THIN, true);
-        BOOST_CHECK(LogAcceptCategory(THIN));
-        LogToggleCategory(THIN, false);
-        BOOST_CHECK(!LogAcceptCategory(THIN));
-        LogToggleCategory(THIN, true);
-        LogToggleCategory(NET, true);
-        BOOST_CHECK(LogAcceptCategory(THIN | NET));
-        LogToggleCategory(ALL, true);
-        BOOST_CHECK_EQUAL(ALL, categoriesEnabled);
-        LogToggleCategory(ALL, false);
-        BOOST_CHECK_EQUAL(NONE, categoriesEnabled);
-        BOOST_CHECK_EQUAL(LogGetLabel(ADDRMAN), "addrman");
-        BOOST_CHECK(TestSetLog(ALL, "all", "on"));
-        BOOST_CHECK(TestSetLog(NONE, "all", "off"));
-        BOOST_CHECK(TestSetLog(NONE, "tor"));
-        BOOST_CHECK(TestSetLog(TOR, "tor", "on"));
-        BOOST_CHECK(TestSetLog(NONE, "tor", "off"));
-        BOOST_CHECK(!TestSetLog(TOR, "tor", "bad-arg"));
-        BOOST_CHECK(TestSetLog(categoriesEnabled, "badcategory", "on"));
-        LogToggleCategory(ALL, true);
-        LOG(THIN, "missing args %s %d\n");
-        LOG(THIN, "wrong order args %s %d\n", 3, "hello");
-        LOG(THIN, "null arg %s\n", NULL);
-        BOOST_CHECK(IsStringTrue("true"));
-        BOOST_CHECK(IsStringTrue("enable"));
-        BOOST_CHECK(IsStringTrue("1"));
-        BOOST_CHECK(IsStringTrue("on"));
-        BOOST_CHECK(!IsStringTrue("false"));
-        BOOST_CHECK(!IsStringTrue("disable"));
-        BOOST_CHECK(!IsStringTrue("0"));
-        BOOST_CHECK(!IsStringTrue("off"));
-        BOOST_CHECK(IsStringTrueBadArgTest("bad"));
-    }
+    using namespace Logging;
```

Comment:
minor nit. this indentation seems unrelated to the addition itself, I wonder if this has been done automatically by clang linter.